### DISCUSSION TITLE
Potential miswiring in PVP_KattenbergOutpost

### DISCUSSION THREAD

**Discussion by 4-S-H** [FakeFishGames/Barotrauma#14943] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Maybe the lever's continuous output should be wired to set_state of the relay component.
Currently, when the lever is pulled, it keeps switching the component on and off every tick.
It does appear to do the intended function to turn off the electricity on one side, but maybe not in the intended way.
![image](https://github.com/user-attachments/assets/c076d9d3-d550-4a1a-8233-61bcad6d61e1)

### Reproduction steps

1. Play PVP_KattenbergOutpost
2. Interact with the lever near the reactor

### Bug prevalence

Just once

### Single player or multiplayer?

Happens outside single player or multiplayer game modes (e.g. game launches on startup, something broken in the main menu)

### Version

v1.6.17.0 (Unto the Breach Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/ba2cce31f9a9a12a82568f7b300d1e0cf46759a0 (commit in our private repo), fix coming up in the next unstable build.
